tests/yam/yast_container/yast2_control_center.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Start containerized YaST Control Center with ncurses
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'basetest';
+use strict;
+use warnings;
+use testapi;
+
+my $podman_cmd = "podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses-test-api";
+
+sub run {
+    select_console('root-console');
+    my $yui_params = get_var('YUI_PARAMS');
+    enter_cmd($yui_params . " " . $podman_cmd);
+    my $control_center = $testapi::distri->get_control_center()->get_control_center_page();
```

Comment:
from the test you only specify the action, not the getter, so here you will call run directly.
You just need to store the variable $control_center to make multiple calls to a couple of modules for example.